src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
@nguerrera showed me this trick. I think it's much better than default(...) as it's independent of property type and works in all members, e.g. methods with out parameter.

@weshaggard